client/my-sites/plans/plan-overview/plan-features/index.jsx

```diff
@@ -81,7 +81,13 @@ const PlanFeatures = React.createClass( {
 					willBeRemoved={ willBeRemoved } />
 
 				<PlanFeature
-					description={ this.translate( 'You can live chat with our happiness engineers anytime you need.' ) }
+					description={
+						isBusiness( this.props.selectedSite.plan )
+						?
+							this.translate( 'You can live chat with our happiness engineers anytime you need.' )
+						:
+							this.translate( 'You can contact our happiness engineers anytime you need.' )
```

Comment:
A few thoughts on the copy:

- I think it should be "any time"
- Should "live chat" be in quotes or Headline Case?
- Should Happiness Engineer be capitalized?